base/docs/helpdb.jl

```diff
@@ -3320,29 +3320,31 @@ Creates an iterable object for visiting each index of an AbstractArray `A` in an
 
 Example for a sparse 2-d array:
 
-    julia> A = sprand(2, 3, 0.5)
-    2x3 sparse matrix with 4 Float64 entries:
-        [1, 1]  =  0.598888
-        [1, 2]  =  0.0230247
-        [1, 3]  =  0.486499
-        [2, 3]  =  0.809041
+```jldoctest
+    julia> A = sparse([1, 1, 2], [1, 3, 1], [1, 2, -5])
+    2x3 sparse matrix with 3 Int64 entries:
+            [1, 1]  =  1
+            [2, 1]  =  -5
+            [1, 3]  =  2
 
     julia> for iter in eachindex(A)
-               @show iter.I_1, iter.I_2
+               @show iter.I[1], iter.I[2]
                @show A[iter]
            end
-    (iter.I_1,iter.I_2) = (1,1)
-    A[iter] = 0.5988881393454597
-    (iter.I_1,iter.I_2) = (2,1)
-    A[iter] = 0.0
-    (iter.I_1,iter.I_2) = (1,2)
-    A[iter] = 0.02302469881746183
-    (iter.I_1,iter.I_2) = (2,2)
-    A[iter] = 0.0
-    (iter.I_1,iter.I_2) = (1,3)
-    A[iter] = 0.4864987874354343
-    (iter.I_1,iter.I_2) = (2,3)
-    A[iter] = 0.8090413606455655
+
```

Comment:
It was! And I never would have noticed that that's the reason it "expected no output", so thanks for the pointer.